src/client/styles/components/_hero.scss

```diff
@@ -64,4 +64,31 @@
       }
     }
   }
+
+  &.staff-picks-hero {
+    background-color: $nypl-red;
+    color: #fff;
+    height: 250px;
+    width: 100%;
+
+    .hero-content-description {
+      background-color: transparent;
+    }
+
+    @include media($mobile-breakpoint) {
+      background-color: #fff;
+      color: #000;
+
+      .hero-section-title {
```

Comment:
Isn't this the same as the visually hidden class rules? Why not use that class?

Reply:
sure!